primitive-types/src/lib.rs

```diff
@@ -14,9 +14,8 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-mod error;
-
-pub use error::{Error, TryFrom, TryInto, Never};
+#![cfg(feature = "std")]
```

Comment:
is this really correct? 
why enforce `feature = std` in the crate? 

I guess it should be `#[cfg(feature = "std")]`?

Reply:
I tested in edition 2015 `#[cfg(feature = "std")]` is still necessary, otherwise compiler would complain `core` being reimported when building no_std.